apps/tailwind-components/components/editor/CodeEditor.vue

```diff
@@ -62,33 +50,33 @@ onMounted(() => {
         <BaseIcon name="CaretDown" :width="24" />
         <span>{{ lang }}</span>
       </button>
-      <!-- <Button
-        type="secondary"
-        icon="UploadFile"
-        :icon-only="true"
-        label="Format code"
-        size="small"
-        class="mx-2"
-        @click="formatEditor"
-      /> -->
     </div>
     <div
+      ref="container"
       :id="`${lang}-editor-content`"
       :class="{
-        block: isExpanded,
+        static: isExpanded,
         hidden: !isExpanded,
       }"
     >
       <MonacoEditor
+        ref="editor"
         v-model="code"
+        @update:model-value="emits('update:modelValue', code as string)"
```

Comment:
maybe 'JSON.stringify' instead of 'as string' to make certain it actually is a string?